berkeley-function-call-leaderboard/bfcl/eval_checker/eval_runner_helper.py

```diff
@@ -244,6 +245,9 @@ def get_cost_letency_info(model_name, cost_data, latency_data):
 def generate_leaderboard_csv(
     leaderboard_table, output_path, eval_models=None, eval_categories=None
 ):
+    # Load metadata at the beginning of the function
+    model_metadata, _, _ = metadata_loader.load_metadata()
+    
```

Comment:
For dynamic load, we'd used `model_metadata` rather than `MODEL_METADATA_MAPPING`